src/backend/base/langflow/services/database/models/transactions/model.py

```diff
@@ -34,9 +34,19 @@ def validate_flow_id(cls, value):
             value = UUID(value)
         return value
 
+    @field_serializer("inputs")
+    def serialize_inputs(self, data) -> dict:
+        truncated_data = truncate_json(data)
+        if isinstance(truncated_data, dict):
+            return truncated_data
+        return {"content": truncated_data}
```

Comment:
Inputs should always be dict
```suggestion
    @field_serializer("inputs")
    def serialize_inputs(self, data) -> dict:
        return truncate_json(data)
```